modules/gatk3/unifiedgenotyper/main.nf

```diff
@@ -0,0 +1,47 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process GATK3_UNIFIEDGENOTYPER {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::gatk=3.5" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/gatk:3.5--hdfd78af_11"
+    } else {
+        container "quay.io/biocontainers/gatk:3.5--hdfd78af_11"
+    }
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path ref
+    path fai
+    path dict
+
+    output:
+    tuple val(meta), path("*.vcf.gz"),        emit: vcf
+    path "versions.yml", emit: versions
+
+    script:
+    def prefix = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    """
+    gatk3 -T UnifiedGenotyper \\
+        -R $ref \\
+        -I $bam \\
+        -o ${prefix}.vcf \\
+        $options.args
+
+    gzip --no-name ${prefix}.vcf
```

Comment:
I just remembered that we prefer to `bgzip` VCF files because this is then compatible downstream with `bcftools

https://github.com/nf-core/eager/blob/cc66639a8a2710f0de6aef5ae28e1614a5a4e3e2/main.nf#L2370

Would be worth updating accordingly (still results in `.gz`, just in a  'block' format)

Reply:
does this require a separate loading on biocontainer? or is bgzip included in default env?